scripts/dts/extract_dts_includes.py

```diff
@@ -275,7 +275,8 @@ def extract_node_include_info(reduced, root_node_address, sub_node_address,
         except KeyError:
             pass
 
-
+    # Extract status
+    default.extract(sub_node_address, yaml, None, None)
```

Comment:
You are adding a special case (prop is None) to default.extract and call it here. This is not very intuitive. 
Why not make it special by creating a status.py and calling this?

Reply:
Sorry, I was planning to answer you on that point, but never clicked on comment.
So, I totally agree with you this is not heuristics so removed from there.
Though, I was reluctant to create a specific class for 2 lines, and I thought default was the good class to use since we're not doing anything specific for this property, even if this require some tuning do the extract function.
In general I'd like we could avoid having people creating class each time a new property shows up. In the other hand, one can argue that it could be even worse if anyone hacks any classes w/o respect to the class.
Let me try to do this with better respect to the class:

In extract_node_include_info:
```
if 'status' in y_node['properties'].keys():
      default.extract(sub_node_address, yaml, 'status', None)
```
Then, in extract() of default class, after edts_insert_device_property(), return if def_label is none to better stick with the default class spirit.

```